### PR TITLE
Added Strats HBK role

### PR DESCRIPTION
**Pull request by rs2**

### CODE REVIEW

File: jobs/hbk-strats-developer.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+contract: permanent
+contact:
+    name: Pav
+    email: pandriychenko@hbk.com
+
+created: !!timestamp '2016-07-21'
+tags:
+  - python
+  - sql
+  - hedge-fund
```
Comment (lordmauve) [inline jobs/hbk-strats-developer.html]: "finance" perhaps?

File: jobs/hbk-strats-developer.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: Strats
```
Comment (lordmauve) [inline jobs/hbk-strats-developer.html]: This should be the full job title.

Comment (rs2) [inline jobs/hbk-strats-developer.html]: It is

Comment (lordmauve) [inline jobs/hbk-strats-developer.html]: That's what would be on a business card? "Pavel Andreichenko, Strats"?

**Pull request merged**